companies/redfin.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Redfin
+slug: redfin
+blm_statements:
+  - url: https://twitter.com/Redfin/status/1267211947973283840?s=20
+    date_posted: 2020-05-31
+  - url: https://twitter.com/Redfin/status/1271141352927969281?s=20
+    date_posted: 2020-06-11
+exec_team_url: [Leadership](https://press.redfin.com/spokespeople)
```

Comment:
```suggestion
exec_team_url: https://press.redfin.com/spokespeople
```

Just the URL here is fine!  We aren't sure where we are displaying this info quite yet.

Reply:
I made the necessary changes